README.rst

```diff
@@ -64,7 +64,7 @@ Life Calendar:
 
   ::
 
-      sudo apt-get install python-cairo
+      sudo apt install libcairo2-dev pkg-config python3-dev
```

Comment:
WHy are pkg-config and python3-dev required?

Reply:
That was the content I saw in SO. I think it's needed for local compilation. Can't check on my system since I already have them installed.